nntrainer/tensor/cpu_backend/ggml_interface/ggml_interface.cpp

```diff
@@ -182,7 +182,38 @@ void __ggml_q4_K_8x8_q8_K_GEMM(const unsigned int M, const unsigned int N,
                                 (void *)((char *)B + M_step_start * B_step),
                                 QA.data(), M, M_step_end - M_step_start);
     }
-  } else { // GEMM
+  }
+  if (M % 4 != 0) {
```

Comment:
Do you intend for the case M == 1 to go through this block as well? 
Should it be an else if instead?

```suggestion
  else if (M % 4 != 0) {
```

Reply:
Woah, my mistakes! thanks!